src/libs/alea/alea.js

```diff
@@ -13,13 +13,13 @@ Modified again by Marcell Perger.
  * @prop {(lo: number, hi: number) => number} randint
  */
 /**
- * @typedef {(RandomNS & () => number)} Random
+ * @typedef {(RandomNS & (() => number))} Random
```

Comment:
This file is marked as generated in `.gitattributes` - should probably not be

Reply:
Or just move it out of this directory. I should to make an issue for this and do a separate pull request.